packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -27,46 +28,39 @@ const useRootStyles = makeStyles({
     width: '6px',
     height: '6px',
     fontSize: '4px',
+    lineHeight: '4px',
   },
   'extra-small': {
     width: '10px',
     height: '10px',
     fontSize: '6px',
+    lineHeight: '6px',
   },
   small: {
     minWidth: '16px',
     height: '16px',
-    ...shorthands.padding('2px'),
-    ...shorthands.gap('4px'),
-    fontSize: '8px',
+    ...shorthands.padding(0, '4px'), // 2px root padding + 2px text padding
```

Comment:
My first thought is that we should be using spacing tokens here but the comment is making me think you intentionally didn't use them.

Is there a reason to not use spacing tokens here? If so, can the comment explicitly state that so we don't update them at a later date? If not, can they be changed to tokens?